Cargo.toml

```diff
@@ -57,6 +57,17 @@ num_cpus = "1.16.0"
 hyper-rustls = "0.24.2"
 rustls = "0.21.9"
 rustls-pemfile = "1.0.4"
+fnv = "1.0.7"
+futures-channel = { version = "0.3.13" }
+futures-timer = { version = "3.0.2" }
+futures-util = { features = [
+  "std",
+  "io",
+  "sink",
+  "async-await",
+  "async-await-macro",
+] }
+lru = { version = "0.12.1" }
```

Comment:
Really? We need all this?

Reply:
we do need `lru` in cache.rs but I'm not sure about those async features will try to compile after removing those

Reply:
I copied the cache code too incase we need it in the future